frontend/src/container/GridGraphLayout/Graph/index.tsx

```diff
@@ -256,7 +257,29 @@ function GridCardGraph({
 					onView={handleOnView}
 					onDelete={handleOnDelete}
 				/>
-				<Spinner height="20vh" tip="Loading..." />
+				{prevChartDataSetRef.current === undefined && (
+					<Spinner
+						height="12vh"
+						tip="Loading..."
+						customStyle={{
```

Comment:
if we are passing static object of styles add this one config and then use is here

Reply:
Done